### PR TITLE
Remove creation of default action plans

### PR DESCRIPTION
**Pull request by andrewmil**
# Motivation and Context
Now that action plans are created at collection exercise creation we don't need to add them before the acceptance tests

# What has changed
Removed unnecessary database calls
Fixed/generalised tests for `display_trading_as` feature
Improved test for `access_eq_collection_exercise` feature

# How to test?
Run with associated PR in collection exercise service
Acceptance tests should still pass

# Links
[Trello](https://trello.com/c/rnSOtuA1/66-create-case-type-default-in-collection-exercise-for-new-survey)
[rm-collection-exercise-service PR](https://github.com/ONSdigital/rm-collection-exercise-service/pull/73)

### CODE REVIEW

**Comment by andrewmil** [comment]
[Create case type default in collection exercise for new survey.](https://trello.com/c/rnSOtuA1/66-create-case-type-default-in-collection-exercise-for-new-survey)

File: set_up_ce_execution.py [hunk 1]
```diff
@@ -1,14 +1,7 @@
-from controllers import collection_instrument_controller, database_controller
+from controllers import collection_instrument_controller
 
 if __name__ == '__main__':
     print("Adding initial data required for executing collection exercises")
-    # Add data required to execute collection exercises
-    database_controller.execute_sql('resources/database/collection_exercise_setup/bricks_201801_setup.sql')
```
[the remark this answers is not in the document]
Comment (andrewmil) [reply set_up_ce_execution.py]: Yep will do that now :+1:

**Comment by gemmai95** [comment]
@benjefferies I don't get the above issue and can't seem to replicate it.

File: acceptance_tests/features/steps/display_trading_as_external.py [hunk 2]
```diff
@@ -12,10 +12,10 @@
 from controllers.party_controller import add_survey, get_party_by_email
 
 
-@given('a company has a separate trading name (s01)')
-def company_has_separate_trading_name_s01(_):
-    _add_survey_for_ru_to_respondent_suppress_exception('example@example.com', '49900000006',
-                                                        _get_last_QBS_collection_exercise_id())
+@given('a company with "{ru_ref}" has a separate trading name')
```
Comment (mfcrocker) [inline acceptance_tests/features/steps/display_trading_as_external.py]: Better wording would be `'a company with the reference "{ru_ref}" has a separate trading name`

**Comment by andrewmil** [comment]
@benjefferies @mfcrocker Think I've found a fix for that test so I'll add it in as part of this PR

**Comment by andrewmil** [comment]
@benjefferies @mfcrocker Fix for that test is in but be aware that if you're running frontstage locally (not in docker) then it also requires the environment variable `EQ_URL=https://eq-test/session?token=` to pass the acceptance tests

**Pull request merged**